sql/branch.sql

```diff
@@ -0,0 +1,29 @@
+BEGIN;
+
+    CREATE TABLE package_managers
+    ( id        bigserial       PRIMARY KEY
+    , name      text            NOT NULL UNIQUE
+     );
+
+    CREATE TABLE packages
+    ( id                    bigserial                   PRIMARY KEY
+    , package_manager_id    bigint                      NOT NULL
+    , name                  text                        NOT NULL
+    , description           text                        NOT NULL DEFAULT ''
+    , long_description      text                        NOT NULL DEFAULT ''
+    , long_description_raw  text                        NOT NULL DEFAULT ''
+    , long_description_type text                        NOT NULL DEFAULT ''
+    , team                  text                        REFERENCES teams
+                                                            ON UPDATE CASCADE ON DELETE RESTRICT
```

Comment:
Let's use `team_id` linking to `id` rather than `slug`, so we don't make even more work for ourselves under https://github.com/gratipay/gratipay.com/issues/835.